config/config.yaml

```diff
@@ -134,7 +134,10 @@ analysis:
         include: true
       # Machine learning analysis (e.g. clustering) of the pathway output files for each dataset
       ml:
+        # ml analysis per dataset
         include: true
+        # required; adds ml analysis per algorithm output
```

Comment:
Maybe this should be optional and default to false?

I haven't looked through the code line-by-line yet. Is the behavior that if this is true, we will get the overall summaries as before and then additionally compute these algorithm-specified summaries? That is what I would expect.